grovedb/src/replication.rs

```diff
@@ -123,6 +120,12 @@ pub fn util_split_global_chunk_id(
         ));
     }
 
+    if global_chunk_id == app_hash {
+        let array_of_zeros: [u8; 32] = [0; 32];
+        let root_chunk_prefix_key: crate::SubtreePrefix = array_of_zeros;
+        return Ok((root_chunk_prefix_key, vec![]));
+    }
+
```

Comment:
In rust you shouldn't have special numbers, meaning that [0;32] shouldn't have a special signicance, instead you should use enums or Options